nibabel/gifti/tests/test_gifti.py

```diff
@@ -126,6 +128,44 @@ def assign_metadata(val):
     pytest.raises(TypeError, assign_metadata, 'not-a-meta')
 
 
+@pytest.mark.parametrize('label', data_type_codes.value_set('label'))
+def test_image_typing(label):
+    dtype = data_type_codes.dtype[label]
+    if dtype == np.void:
+        return
+    arr = 127 * rng.random(
+        20,
+    )
```

Comment:
This could be one line I guess :slightly_smiling_face:

Reply:
```suggestion
    arr = 127 * rng.random(20)
```